soil_params_Gupta2020_2022/transform_geotiff_to_netcdf.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Define the source and destination directories
+SRC_DIR="/groups/esm/ClimaArtifacts/artifacts/soil_params_Gupta2020_2022/raw_data"
+DEST_DIR="/groups/esm/ClimaArtifacts/artifacts/soil_params_Gupta2020_2022/raw_data_nc"
```

Comment:
We should avoid hard-coding absolute paths here

Reply:
I removed this. but the user would need to fill in the paths for their local machine for this part. I left a comment.

Reply:
You can use relative paths, like
```
SRC_DIR="soil_params_Gupta2020_2022/raw_data"
DEST_DIR="soil_params_Gupta2020_2022/raw_data_nc"
```
(or whatever is correct)

Reply:
Im not sure we know this - once they download the raw data from the zenodo source, it will be in their downloads folder on their local machine. the naming "soil_params_Gupta2020_2022" doesnt exist for them at that point

Reply:
Okay, you can leave the blank